Doc/library/os.rst

```diff
@@ -3195,6 +3195,27 @@ features:
    .. versionadded:: 3.8
 
 
+.. function:: eventfd(initval[, flags=os.EFD_CLOEXEC])
+
+   Create an event file descriptor
```

Comment:
I think this could be expanded upon a bit, such as:

1) Explicitly mention that the file descriptor is returned. This is done for the above `memfd_create`.
2) Briefly summarize the use for it as a lightweight notifier/waiter for events. Compared to a pipe, it is more resource efficient as an event notifier.

E.g.

```suggestion
   Create and return an "eventfd" file descriptor. It can be used as a lightweight
   event notifier.
   
   .. note::
      Compared to a pipe that is exclusively used for event notification,
      `eventfd()` can be used as a more resource efficient alternative that
      only uses a single file descriptor and has far less overhead.
```

While I agree with making the low-level OS functions that are mappings to existing OS APIs fairly minimalist in terms to documentation details (especially if they're already well documented), I think the existing one could provide a bit more information to the user that explains *why* they might want to use it. 

For full details one can view the man page, but IMO, we at least should summarize the general purpose. Otherwise, I suspect it will significantly deter discoverability of the feature.